internal/rukpakctl/run.go

```diff
@@ -67,27 +59,17 @@ func (r *Run) Run(ctx context.Context, bundleDeploymentName string, bundle fs.FS
 		return false, err
 	}
 
-	bundleLabels := map[string]string{
+	bundleDeploymentLabels := map[string]string{
```

Comment:
The fact that the BD had a bundle template with more metadata was important for this upload use case, specifically for the use case of uploading new contents for an existing BD to pivot from the old bundle content to the new bundle content).

Whenever an upload happens, the client gets a digest of the uploaded data and includes that in the bundle template metadata. That way, when the BD reconciles, it sees that the bundle template is different and creates a new Bundle object, which allows the upload web service to accept an HTTP POST for the new data.

Now, it seems like we change the BDs labels, but in the case that the BD is already unpacked, the upload manager will fail to upload new content to it.

We likely need a few more tweaks here to make things work without an underlying Bundle API.